### DISCUSSION TITLE
RavenDB backup to network path - error

### DISCUSSION THREAD

**Discussion by audunsaltvik** [ravendb/ravendb#15152] [General]
Hi!
I'm trying to setup a backup shedule with a network path as backup directory. I have done this on other RavenDB servers earlier, but this one won't accept the path:
`Error: Path \\servername\directory cannot be accessed because '\\servername\directory' doesn't exist`

I'm able to access the share from Windows and add stuff to it. Username/password is not required on the share. TCP ports 445 and 139 is opened through firewall.

Any ideas of what's missing to be able to access the share from RavenDB?

**Comment by ayende** [comment]
What user is the RavenDB process running under?
The usual reason for this is that the RavenDB process is running under something like `Local Service`, which doesn't have the permissions to access network partitions.

**Reply by audunsaltvik** [reply]
Thank you, running the service under a domain user worked perfectly.
